src/sentry/runner/commands/execfile.py

```diff
@@ -0,0 +1,43 @@
+import pathlib
+import sys
+
+import click
+
+
+@click.command(
+    name="execfile", context_settings=dict(ignore_unknown_options=True, allow_extra_args=True)
+)
+@click.argument("filename", required=True)
+def execfile(filename):
+    """Execute a script.
+
+    This is very similar to `exec`, with the following differences:
```

Comment:
I guess I understand the differences but I don't really know why you can't use `sentry exec`?

Reply:
The real blocker is the missing `__file__`.  The fact that it mutates the code from the file and all the line numbers and source code reporting form tracebacks is useless is a strong second.

Reply:
I think it would be reasonably safe to fix that in `sentry exec`, afaict it's not used in production at all

Reply:
Yeah, it crossed my mind but I wasn't sure how much liberty I could take with the existing `exec` command so went for the safe option.  Is it acceptable to merge this and follow up with unifying the two back into `exec` so that can have a long review cycle with more people acking it?